cmd/ketch/main.go

```diff
@@ -28,8 +28,24 @@ func main() {
 		log.Fatalf("couldn't create pack service %q", err)
 	}
 
-	cmd := newRootCmd(&configuration.Configuration{}, out, packSvc)
+	cmd := newRootCmd(initConfig(), out, packSvc)
 	if err := cmd.Execute(); err != nil {
 		log.Fatalf("execution failed %q", err)
 	}
 }
+
+func initConfig() *configuration.Configuration {
+	path, err := configuration.DefaultConfigPath()
+	if err != nil {
+		log.Println(err)
+		return &configuration.Configuration{}
+	}
+
+	cfg, err := configuration.Read(path)
```

Comment:
I am not sure but do we create that path if it is not already present?

Reply:
The path is not created if it doesn't already exist. The config file it is looking for is not necessary for ketch to run, so in the event it is not found a configuration is returned populated with the zero value for `AdditionalBuilders`, which is an empty array of strings. Since there is nothing in the array, the list argument will not append any additional builders to the default list.

Reply:
The new builders added by the user are specified by them in a config.toml file. A user can use a builder not in the list and it will work (build an image using that builder), but the builder details will not be added to the list.